src/Sylius/Bundle/ApiBundle/Resources/config/services/extensions.xml

```diff
@@ -38,6 +38,11 @@
             <tag name="api_platform.doctrine.orm.query_extension.collection" />
         </service>
 
+        <service id="Sylius\Bundle\ApiBundle\Doctrine\QueryCollectionExtension\OrdersByChannelExtension">
+            <argument type="service" id="Sylius\Bundle\ApiBundle\Context\UserContextInterface" />
+            <tag name="api_platform.doctrine.orm.query_extension.collection" />
```

Comment:
Maybe we could tag all the services implementing `ContextAwareQueryCollectionExtensionInterface` with `api_platform.doctrine.orm.query_extension.collection` [automatically](https://symfony.com/doc/current/service_container/tags.html#autoconfiguring-tags) 🚀

Reply:
:+1:
It will go in a separate PR.
Most likely more services can be auto-tagged this way

Reply:
Indeed :D 🚀